exercise_2/main.py

```diff
@@ -0,0 +1,20 @@
+import sys
+from openweather_api_client import BasicWeatherApiClient
+
+
+if __name__ == "__main__":
+    try:
+        weather = BasicWeatherApiClient()
+        complete_weather = weather.get_weather()
+        current_weather = weather.parse_weather(complete_weather)
+    except KeyError:
+        print("City not found.")
+        sys.exit()
```

Comment:
Nie musisz się martwić narazie takimi zamykaniami programów. Będziemy to robić imaczej i przy użyciu fajnych narzędzi :)